tests/exports/factories.py

```diff
@@ -0,0 +1,47 @@
+import factory
+
+from adhocracy4.test import factories as a4_factories
+from apps.budgeting import models as budgeting_models
+from apps.moderatorfeedback import models as moderatorfeedback_models
+from tests import factories
+from tests.ideas import factories as idea_factories
+
+
+class RatingFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = 'a4ratings.Rating'
+
+    value = 1
+    creator = factory.SubFactory(a4_factories.USER_FACTORY)
+    content_object = factory.SubFactory(idea_factories.IdeaFactory)
+
+
+class ModeratorStatementFactory(factory.django.DjangoModelFactory):
+
+    class Meta:
+        model = moderatorfeedback_models.ModeratorStatement
+
+    statement = factory.Faker('text')
+    creator = factory.SubFactory(factories.UserFactory)
```

Comment:
Maybe we should move the general factories (Rarting, ModeratorStatement) to test/factories.py?
Otherwise we might end up duplicating them in the next app.